optaplanner-docs/src/main/asciidoc/DesignPatterns/DesignPatterns-chapter.adoc

```diff
@@ -9,92 +9,74 @@
 
 
 [[designPatternsIntroduction]]
-== Design Patterns Introduction
-
-These design patterns list and solve common design challenges.
+== Design patterns introduction
 
+The design patterns in this section list and solve common design challenges.
+//Need to explain what design patterns are.
```

Comment:
"Design patterns" is a term introduced by the Gang of Four (including Eric Gamma) book
Since then, there are many similar books.

"Design patterns" are normally part of every programming education, so I'd argue it's a known term, but it doesn't hurt to give a one line explanation. See also https://en.wikipedia.org/wiki/Design_Patterns

How about:
"OptaPlanner Design patterns are general, reusable solutions to common challenges in the model or architecture of projects that do constraint solving."

Reply:
Added.